### PR TITLE
Put docker-compose file to each benchmark directory

### PR DESCRIPTION
**Pull request by kou**
If we put docker-compose file to each benchmark directory, we can use
the standard name, docker-compose.yml.

Signed-off-by: Sutou Kouhei <kou@clear-code.com>

### CODE REVIEW

File: benchmark.sh [hunk 1]
```diff
@@ -7,35 +7,43 @@ DURATION=120
 [ "$(uname -s)" == "Darwin" ] && SED=gsed
 [ "$(uname -s)" == "Darwin" ] && HEAD=ghead
 
+BENCHMARK_NAME="$1"
+if [ -f "${BENCHMARK_NAME}.yml" ]; then
+  DOCKER_COMPOSE_YML="${BENCHMARK_NAME}.yml"
```
Comment (ganmacs) [inline benchmark.sh]: Is this condition needed? All docker-compose.yml files seems to be moved into each directory.

File: benchmark.sh [hunk 2]
```diff
@@ -7,35 +7,43 @@ DURATION=120
...
+if [ -z "${DOCKER_COMPOSE_YML}" ] ; then
   echo Usage:
   echo
   echo "  $0 testsuite"
-  echo 
+  echo
   echo Available testsuites:
   echo
-  find -- * -name \*.yml | $SED -e 's/\.yml$//' | sort
+  ls */docker-compose.yml | $SED -e 's,/docker-compose.yml$,,' | sort | uniq
```
Comment (ganmacs) [inline benchmark.sh]: Does a file system guarantee the uniqueness of these values? ( can we trim `uniq`? )

**Review by ganmacs — APPROVED** [review]
LGTM with nits comments

File: benchmark.sh (hunk 1 above)
Comment (kou) [reply benchmark.sh]: Oh, sorry. I should have removed this.
I tried keeping backward compatibility first. But I decided that the backward compatibility is needless.

I'll remove this.

File: benchmark.sh (hunk 2 above)
Comment (kou) [reply benchmark.sh]: Sorry. This is needless.
It's needed if we keep backward compatibility.

I'll remove this too.

**Review by kou — COMMENTED** [review]
I'll fix them.
Sorry for leaving some needless codes that are only needed when I was working on this.

**Comment by kou** [comment]
Done.

**Comment by ganmacs** [comment]
Thank you 🙏

**Pull request merged**